### PR TITLE
Changed Image positions and dimensions and added border to image

### PR DESCRIPTION
**Pull request by 96RadhikaJadhav**
Changed Image positions and dimensions and added border to image in flow_sequence_diag.md

This PR will resolve issue #11 .

### CODE REVIEW

File: DBMS module/flow_sequence_diag.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # Data Flow Diagram
 
-<img width="700" height="535" src="/images/flow-sequensce.png"> 
+<img width="650" height="700" src="/images/flow-sequensce.png" alt="Flow sequence diagram" border= "7" style= "display: block; margin-left: auto; margin-right: auto; ">
```
Comment (IIITM-Jay) [inline DBMS module/flow_sequence_diag.md:3]: @96RadhikaJadhav I think the border is not yet set....
Is the border visible to you,... 
if yes, then can you attach the screenshot here.

**Review by IIITM-Jay — CHANGES_REQUESTED** [review]
no border is visible ....
Try to fix it, if you are facing any problem then feel free to share it here

**Comment by IIITM-Jay** [comment]
@96RadhikaJadhav Thanks for the first PR... :1st_place_medal: :tada: :champagne: :sparkles:
There is no hurry , you can complete this as your own conviniency level and sure we can fix those and this is the platform where you can open your thoughts. :+1:

**Comment by 96RadhikaJadhav** [comment]
> @96RadhikaJadhav Thanks for the first PR... 🥇 🎉 🍾 ✨
> There is no hurry , you can complete this as your own conviniency level and sure we can fix those and this is the platform where you can open your thoughts. 👍

@IIITM-Jay  While making changes I have opened file in atom editor and the border was visible there. But now when I tried opening the file through chrome, the image isn't visible to me so can't see if the border is there or not? Any suggestions how can I fix this issue?

**Comment by IIITM-Jay** [comment]
@96RadhikaJadhav I have open up one discussion #25 , go through it deliver your ideas there and also encourage your friends, juniors and colleagues also to join this repo and team.. :+1:

**Comment by IIITM-Jay** [comment]
> > @96RadhikaJadhav Thanks for the first PR... 1st_place_medal tada champagne sparkles
> > There is no hurry , you can complete this as your own conviniency level and sure we can fix those and this is the platform where you can open your thoughts. +1
> 
> @IIITM-Jay While making changes I have opened file in atom editor and the border was visible there. But now when I tried opening the file through chrome, the image isn't visible to me so can't see if the border is there or not? Any suggestions how can I fix this issue?

I will look tomorrow regarding this...

**Comment by 96RadhikaJadhav** [comment]
> > > @96RadhikaJadhav Thanks for the first PR... 1st_place_medal tada champagne sparkles
> > > There is no hurry , you can complete this as your own conviniency level and sure we can fix those and this is the platform where you can open your thoughts. +1
> > 
> > 
> > @IIITM-Jay While making changes I have opened file in atom editor and the border was visible there. But now when I tried opening the file through chrome, the image isn't visible to me so can't see if the border is there or not? Any suggestions how can I fix this issue?
> 
> I will look tomorrow regarding this...

Sure :) .

**Comment by 96RadhikaJadhav** [comment]
> > > > @96RadhikaJadhav Thanks for the first PR... 1st_place_medal tada champagne sparkles
> > > > There is no hurry , you can complete this as your own conviniency level and sure we can fix those and this is the platform where you can open your thoughts. +1
> > > 
> > > 
> > > @IIITM-Jay While making changes I have opened file in atom editor and the border was visible there. But now when I tried opening the file through chrome, the image isn't visible to me so can't see if the border is there or not? Any suggestions how can I fix this issue?
> > 
> > 
> > I will look tomorrow regarding this...
> 
> Sure :) .

image

**Comment by 96RadhikaJadhav** [comment]
> > > > > @96RadhikaJadhav Thanks for the first PR... 1st_place_medal tada champagne sparkles
> > > > > There is no hurry , you can complete this as your own conviniency level and sure we can fix those and this is the platform where you can open your thoughts. +1
> > > > 
> > > > 
> > > > @IIITM-Jay While making changes I have opened file in atom editor and the border was visible there. But now when I tried opening the file through chrome, the image isn't visible to me so can't see if the border is there or not? Any suggestions how can I fix this issue?
> > > 
> > > 
> > > I will look tomorrow regarding this...
> > 
> > 
> > Sure :) .
> 
> image

image
 And this one is for atom editor showing border

**Comment by 96RadhikaJadhav** [comment]
> > > > > > @96RadhikaJadhav Thanks for the first PR... 1st_place_medal tada champagne sparkles
> > > > > > There is no hurry , you can complete this as your own conviniency level and sure we can fix those and this is the platform where you can open your thoughts. +1
> > > > > 
> > > > > 
> > > > > @IIITM-Jay While making changes I have opened file in atom editor and the border was visible there. But now when I tried opening the file through chrome, the image isn't visible to me so can't see if the border is there or not? Any suggestions how can I fix this issue?
> > > > 
> > > > 
> > > > I will look tomorrow regarding this...
> > > 
> > > 
> > > Sure :) .
> > 
> > 
> > image
> 
> image
> And this one is for atom editor showing border

Hi @IIITM-Jay   Did you get any time to check on this?

**Comment by 96RadhikaJadhav** [comment]
> Hi @IIITM-Jay Did you get any time to check on this?

Meanwhile I have tried using different markdown extension in chrome. But here it's still not showing image but the border is visible.

image

**Comment by 96RadhikaJadhav** [comment]
> > Hi @IIITM-Jay Did you get any time to check on this?
 
The image along with border is now visible. Please see below image.

image

Please share your thoughts on this and let me know if it's good to be merged, so that I can take up another issue in this project :) .

**Comment by IIITM-Jay** [comment]
@96RadhikaJadhav I must think that you have'nt updated the PR and perhaphs you have made changes to your local repo that you have cloned in your PC :worried: :disappointed: :confused: 

One more important thing I would like to share with you that as far as, there is a continuous flow of the conversation there is no need to quote the reply again and again, as it would be then difficult (must guess you also foud it ugly and difficult) for searching the exact new thing that you are sharing, by saying this I would like to say that quote the reply when there is a misunderstanding of the communication or any flaw occur in the consistency and whenever you feel necessary.

**Comment by IIITM-Jay** [comment]
@96RadhikaJadhav Radhika , I would suggest you to please push the updated file with border specifications once again ....

**Comment by 96RadhikaJadhav** [comment]
> @96RadhikaJadhav I must think that you have'nt updated the PR and perhaphs you have made changes to your local repo that you have cloned in your PC 😟 😞 😕
> 
> One more important thing I would like to share with you that as far as, there is a continuous flow of the conversation there is no need to quote the reply again and again, as it would be then difficult (must guess you also found it ugly and difficult) for searching the exact new thing that you are sharing, by saying this I would like to say that quote the reply when there is a misunderstanding of the communication or any flaw occur in the consistency and whenever you feel necessary.

Yeah sorry for that! I noticed that after replying to messages 😞. Will take care of it further.

**Comment by 96RadhikaJadhav** [comment]
@IIITM-Jay  There is a lot of confusion for me now 😞. While pushing I'm getting error like Error: failed to push some refs. I did some searching and tried to pushed it again but its still not getting pushed 😟.
If you allow can I create a fresh PR including all image dimensions and position details along with border.

**Comment by IIITM-Jay** [comment]
@96RadhikaJadhav I would suggest you to go through once here :point_down: 
[github-git error:failed](https://stackoverflow.com/questions/24114676/git-error-failed-to-push-some-refs-to-remote)

**Comment by 96RadhikaJadhav** [comment]
@IIITM-Jay  I have already gone through the same link that you've provided. But still getting error. If you don't mind can you please provide your mail id, need your assistance on the issue.

**Comment by 96RadhikaJadhav** [comment]
> @IIITM-Jay I have already gone through the same link that you've provided. But still getting error. If you don't mind can you please provide your mail id, need your assistance on the issue.

@IIITM-Jay  Meanwhile I have tried all other ways regarding the issue I'm facing. But it's not getting committed. If you are busy, can we just ignore this PR and will create a new one. 
If not, please provide your mail id, we can discuss the issue there.

**Comment by IIITM-Jay** [comment]
@96RadhikaJadhav I think you havn't tried by editing the previous PR and then push it online without the use CLI... I think it should work.
**Steps:**
1. Go to the online repository at github.
2. Open up your PR there.
3. Go to the file you have pushed in the repective PR.
4. At the right corner you will find *edit the file* button
5. Click on it and then do the necessary changes and then it will ask you for the push button.

After doing this, let me know if you are not able to do it or facing any such difficulties.
Actually I am on the journey that's why it's taking a lot of time to reply... kindly ignore those :bowing_man:

**Comment by 96RadhikaJadhav** [comment]
> @96RadhikaJadhav I think you havn't tried by editing the previous PR and then push it online without the use CLI... I think it should work.
> **Steps:**
> 
> 1. Go to the online repository at github.
> 2. Open up your PR there.
> 3. Go to the file you have pushed in the repective PR.
> 4. At the right corner you will find _edit the file_ button
> 5. Click on it and then do the necessary changes and then it will ask you for the push button.
> 
> After doing this, let me know if you are not able to do it or facing any such difficulties.
> Actually I am on the journey that's why it's taking a lot of time to reply... kindly ignore those 🙇‍♂️

@IIITM-Jay  That was super easy. Thank you :) . I committed changes. Please see if the border is visible to you now. Also please let me know if any other changes required.

**Comment by IIITM-Jay** [comment]
@96RadhikaJadhav I think now also not visible... see once :disappointed:

**Comment by 96RadhikaJadhav** [comment]
> @96RadhikaJadhav I think now also not visible... see once 😞

Ohh no..But it is visible to me both in editor as well as chrome. Not sure why it's not visible to you 😞.

**Comment by 96RadhikaJadhav** [comment]
@IIITM-Jay Leave it then 😞..Don't merge the PR.

**Comment by 96RadhikaJadhav** [comment]
@IIITM-Jay  So can I work on other issues in this repository??

**Comment by IIITM-Jay** [comment]
@96RadhikaJadhav why you are giving up in so much hurry. you can explore more on flavored markdown and css/ html that's the only solution to this issue. It's not too late.

Are you a part of any winter of code, or preparing for any other platforms? :thinking:

**Comment by 96RadhikaJadhav** [comment]
> @96RadhikaJadhav why you are giving up in so much hurry. you can explore more on flavored markdown and css/ html that's the only solution to this issue. It's not too late.
> 
> Are you a part of any winter of code, or preparing for any other platforms? 🤔

No, I'm not part of any of such programs. Sure, I will do more research on it and will get back to you.

**Comment by 96RadhikaJadhav** [comment]
image


@IIITM-Jay  As per the new changes, the border is as seen in above screenshot.  Please let me know if the changes are now visible to you or not.

**Comment by IIITM-Jay** [comment]
@96RadhikaJadhav for me it's showing grey boundary only and not the black as you posted the screenshot.
You can view your file too that you pushed at the right corner you will find the option to view the file.

Because I must guess that you are viewing with your local repo and it's showing perfectly there but when you will view on the online repo it's not .

Just check it by yourself and let me know

File: DBMS module/flow_sequence_diag.md [hunk 2]
```diff
@@ -1,7 +1,7 @@
 # Data Flow Diagram
 
 
-<img width="650" height="700" src="../images/flow-sequensce.png" alt="Flow sequence diagram" border= "7" style= "display: block; margin-left: auto; margin-right: auto; ">
+<kbd><img width="650" height="700" src="../images/flow-sequensce.png" alt="Flow sequence diagram" style="border: 4px solid black" style= "display: block; margin-left: auto; margin-right: auto; "></kbd>
```
Comment (IIITM-Jay) [inline DBMS module/flow_sequence_diag.md:4]: @96RadhikaJadhav May I know the significance of using ```<kbd>``` here :thinking:

**Review by IIITM-Jay — COMMENTED** [review]
Screenshot from 2021-01-21 10-43-38
I have left one query @96RadhikaJadhav , let me know .... and also attaching the screenshot os what is visible based on your changes

File: DBMS module/flow_sequence_diag.md (hunk 2 above)
Comment (96RadhikaJadhav) [reply DBMS module/flow_sequence_diag.md:4]: When I searched it on stack overflow, in one of the answers I found this tag. So tried applying it.

**Comment by 96RadhikaJadhav** [comment]
@IIITM-Jay  I have tried several commits and viewed that file in online repo. It's same. Tried removing <kbd> tag, but then, no border is visible. I guess one more way to resolve the issue is I can remove the existing image file and can add new image file with the border added to it. Please let me know your thoughts on this.
Or else if you say, I will do more research and will try to come up with some new changes to make border visible.

**Comment by IIITM-Jay** [comment]
@96RadhikaJadhav Radhika if you wish and want to try the above method, feel free to try and see the effect ... 🍾

**Comment by IIITM-Jay** [comment]
@96RadhikaJadhav Sorry :disappointed:  Radhika for letting you wait for so long... its ready to get merged... although the image you uploaded with black boundary is visible as gray but still I appreciate :1st_place_medal: your harwork and efforts :champagne: Congratulations for being a part of this repository. Thank you so much... :tada: :sparkles: :sparkler: :100:

**Comment by IIITM-Jay** [comment]
@all-contributors please add @96RadhikaJadhav for documentation

**Comment by IIITM-Jay** [comment]
@all-contributors please add @96RadhikaJadhav for design

**Comment by IIITM-Jay** [comment]
@all-contributors please add @96RadhikaJadhav for design

**Pull request merged**